sanctions/apps/sanctions/models.py

```diff
@@ -0,0 +1,83 @@
+from simple_history.models import HistoricalRecords
+
+from django.core.validators import MinLengthValidator
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+
+class SanctionsCheckFailure(TimeStampedModel):
+    """ Record of SDN and ISN check failure. """
+    history = HistoricalRecords()
+    full_name = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
+    lms_user_id = models.IntegerField(null=True, db_index=True)
+    city = models.CharField(max_length=32, default='')
+    country = models.CharField(max_length=2)
+    site = models.ForeignKey('sites.Site', verbose_name=_('Site'), null=True, blank=True, on_delete=models.SET_NULL)
+    sanctions_type = models.CharField(max_length=255)
```

Comment:
The idea here is that if in the future we have another type of sanctions (and if it would hypothetically use this same model) we could differentiate here.